scripts/modify_lines.py

```diff
@@ -64,5 +71,8 @@ def limit_line_capacities(n, boi, max_limit):
 
     limit_line_capacities(n, boi, max_limit)
 
+    if snakemake.config.get('new_line', True):
```

Comment:
```suggestion
    if snakemake.config.get('new_line', False):
```

I think the default should be not adding the line, no? In all other configs this entry doesn't exist